lib/matplotlib/_mathtext.py

```diff
@@ -2723,6 +2727,9 @@ def subsuper(self, s, loc, toks):
                 shift_down = lc_baseline + constants.subdrop * xHeight
             else:
                 shift_down = constants.sub1 * xHeight
+                if s.startswith(r'\left', loc):
+                    shift_down = max(
+                        shift_down, nucleus.depth + capHeight / 10)
```

Comment:
codecov thinks this path is not triggered?

Reply:
Not sure on this, but last I checked, we don't have any image comparison tests of the form `\left(\dfrac{N}{D}\right)_S^S`.